PWG/EMCAL/EMCALtasks/AliEmcalCorrectionCellEnergy.cxx

```diff
@@ -221,6 +221,19 @@ Int_t AliEmcalCorrectionCellEnergy::InitRunDepRecalib()
 {
   if (!fEventManager.InputEvent())
     return 0;
+
+  // MF Disable temperature calibration for run2 for the moment
+  // A memory leak in the AliOADBContainer was observed when the 
+  // container is deleted. As consequence when processing multiple
+  // runs the correction task leaks ~100 MB per run included in the 
+  // job, where 70% of the memory leak comes from the temperature
+  // calibration. Untill a fix is provided the temperature calibration
+  // has to be disabled for run2 runs. Disabling has to be done before
+  // opening the OADB container
```

Comment:
Hi @mfasDa , I'm just looking at the code now - could you add the link here too?

Reply:
Done

Reply:
Hello @raymondEhlers,
yes, this is a meta dataset with multiple correction tasks and therefore, of course, the worst case. 
Here is also an example of a meta dataset, but with the Tender: http://alitrain.cern.ch/train-workdir/PWGGA/GA_pPb/719_20180212-1953/test/__ALL__/syswatch.png
This shows the same behaviour.
Cheers, Nicolas

Reply:
Thanks @mfasDa